packages/common/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "devDependencies": {
     "@types/lodash": "4.17.5",
+    "@types/node": "catalog:",
```

Comment:
just to check if this needed?

Reply:
👀  odd i'm checking on master but i don't see this package in common/package.json
<img width="298" height="239" alt="image" src="https://github.com/user-attachments/assets/0bc3d56b-2be1-46d2-a6e4-423392de6f29" />

maybe we can just remove this then if that's the case